lms/templates/discussion/_underscore_templates.html

```diff
@@ -1,608 +1,19 @@
 <%namespace name='static' file='../static_content.html'/>
-<%!
-from django.utils.translation import ugettext as _
-from django.template.defaultfilters import escapejs
-from django_comment_client.permissions import has_permission
-%>
+<%! import json %>
 
-## IMPORTANT: In order to keep js tests valid and relevant, please be sure to update the appropriate HTML in
-## common/static/coffee/spec/discussion_spec_helper.coffee is changed and regenerated, whenever this one changes.
```

Comment:
The Jasmine tests need to be updated to use the new underscore templates, or at the very least discussion_spec_helper.coffee (which is actually in common/static/coffee/spec/discussion/) needs to be updated to contain the new template code. If you go the latter route, then this comment should be restored.